drivers/ti/uart/16550_console.S

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Can you not reuse the top level console API from `drivers/console/console.S` and implement the core functions here i.e. similar to the split between `console.S` and `drivers/arm/pl011/pl011_console.S`?